### ISSUE TITLE
Integrate the Horizontal Rule styled component

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#852]
**Is your feature request related to a problem? Please describe.**
Currently the [code](https://github.com/BBC-News/simorgh/commit/23baba99672b000d46629d697ee0e48d50560bcb?diff=unified#diff-10a1331794c1480312f306da85ffc1a4R11) uses the default html element `<hr>`. We have a `HorizontalRule` styled component which we should integrate.

**Describe the solution you'd like**
Integrate the `HorizontalRule` styled component.

**Describe alternatives you've considered**
N/A

**Testing notes**
Will Cypress tests be required or are unit tests sufficient?

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

**Comment by ChrisBAshton** [comment]
While we're here, we should remove the [Text dependency from HorizontalRule's Storybook](https://github.com/BBC-News/simorgh/blob/latest/src/app/components/HorizontalRule/index.stories.jsx#L4) - it's only used to help demonstrate the `<hr>` in context.

Text component disappears in #1033 so we can't rely on it from here on.

*EDIT: oh, it's already being removed in https://github.com/BBC-News/simorgh/pull/1033/files#diff-d8f6746739a1d3a4cfa9ea223589899eR4*

**Comment by ChrisBAshton** [comment]
Blocked on #1033.

**Comment by ChrisBAshton** [comment]
Superseded by https://github.com/BBC-News/simorgh/issues/1061.
